requirements.txt

```diff
@@ -13,7 +13,7 @@ jedi==0.13.1
 lazy-object-proxy==1.3.1
 mccabe==0.6.1
 more-itertools==8.3.0
-numpy==1.22.0
+numpy==1.19.5
```

Comment:
@msaudulhassan  is this going to be an issue? downgrading numpy?

Reply:
Hi Alex, it is still a draft now. Maybe I should merge develop to my branch, solve the conflict and then reopen the PR.